.claude/commands/impl/rest-api.md

```
@@ -0,0 +1,410 @@
+
+## 사용법
+
+```bash
+/impl rest-api {feature-name}
+/impl rest-api order-cancel
+/impl rest-api member-register
+```
+
+## 실행 프로세스
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)